recipes/libjxl/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class LibjxlConan(ConanFile):
+    name = "libjxl"
+    description = "JPEG XL image format reference implementation"
+    license = "Apache-2.0"
```

Comment:
I think it's BSD-3-Clause:
https://github.com/libjxl/libjxl/blob/main/LICENSE
https://opensource.org/licenses/BSD-3-Clause
```suggestion
    license = "BSD-3-Clause"
```